app/(tabs)/classes.tsx

```diff
@@ -1,11 +1,76 @@
-import { View, Text } from "react-native";
+import React from 'react';
+import { View, ScrollView, StyleSheet } from 'react-native';
+import Colors from '../utils/Colors';
+import ClassCard from '../components/classCard';
+import { time } from 'console';
 
 const ClassesPage = () => {
+    const classes = [
+        {
+            time: 'Dienstag, 13:00',
+            title: 'Mathematik',
+            members: '3 von 10 Plätzen belegt',
+            year: '5. Klasse',
+            status: 'Öffenlich',         
+            imageUrl: 'https://picsum.photos/200',
+        },
+        {
+            time: 'Mittwoch, 14:00',
+            title: 'Physik',
+            members: '3 von 10 Plätzen belegt',
+            year: '6. Klasse',
+            status: 'Privat',
+            imageUrl: 'https://picsum.photos/200',
+        },
+        {
+            time: 'Donnerstag, 15:00',
+            title: 'Chemie',
+            members: '3 von 10 Plätzen belegt',
+            year: '7. Klasse',
+            status: 'Öffenlich',
+            imageUrl: 'https://picsum.photos/200',
+        },
+        {
+            time: 'Freitag, 16:00',
+            title: 'Biologie',
+            members: '3 von 10 Plätzen belegt',
+            year: '8. Klasse',
+            status: 'Privat',            
+            imageUrl: 'https://picsum.photos/200',
+        },
+    ];
+
     return (
-        <View>
-            <Text>Classes Page</Text>
+        <View style={styles.container}>
+            <ScrollView style={styles.list}>
+                <View>
+                    {classes.map((course, index) => (
+                        <ClassCard
+                            key={index}
+                            time={course.time}
+                            title={course.title}
+                            members={course.members}
+                            year={course.year}
+                            status={course.status}
+                            imageUrl={course.imageUrl}
+                            onPress={() => { }}
+                        />
+                    ))}
+                </View>
+            </ScrollView>
         </View>
-    )
+    );
 };
 
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: Colors.background,
+    },
+    list: {
+        flex: 1,
+        marginTop: 10,
```

Comment:
You can simply add a margin for on all for sides using ``margin: N`` and the you don't need the ``marginHorizontal: 30` in the ``classCard`` component styles.

Also, Nils used a margin of ``16`` so let's stick to that for a more seemless design.

Reply:
done